plugins/kubernetes/app/assets/javascripts/kubernetes/controllers/kubernetes_release_wizard_ctrl.js

```diff
@@ -0,0 +1,31 @@
+samson.controller('KubernetesReleaseWizardCtrl', function($scope) {
+  $scope.steps = [
+    {
+      step: 1,
+      current: false
+    },
+    {
+      step: 2,
+      current: false
+    }
+  ];
+
+  $scope.next = function() {
+    showStep($scope.currentStep + 1);
+  };
+
+  $scope.previous = function() {
+    showStep($scope.currentStep - 1);
+  };
+
+  function showStep(step) {
+    _.findWhere($scope.steps, {step: step}).current = true;
```

Comment:
I can replace it by `$scope.steps[step - 1].current`.

The reason why I went with this approach was to make it safer, i.e., not being dependent on the order those steps are declared. Regarding the performance, there's just 2 steps in there, so, it wouldn't become a problem.